app/controllers/api/v1/user_availabilities_controller.rb

```diff
@@ -18,7 +18,11 @@ def destroy
 
   def multiple
     ids = time_block_ids_params
-    ids.each {|id| UserAvailability.create(user: current_user, time_block_id: id) }
+    availabilities = ids.map { |id| UserAvailability.create(user: current_user, time_block_id: id) }
+    if User.find_available >= User.all.size * 0.8 && !Company.first.reminder_sent?
+      ReminderSender.new.call
+    end
+    render json: availabilities
```

Comment:
it's still good habit to return info, as it can help with debugging
if I return 302 (which is done by default) then from the network tab I don'ť know if the data I wanted to implement was successfully implemented or not